### ISSUE TITLE
Accessibility team: How should VoiceOver rotor work with navigating transcript?

### ISSUE THREAD

**Issue by corinagum** [microsoft/BotFramework-WebChat#3759]
BUG: note that for mobile, 'Use arrow keys...' aria-label at End of transcript should not be spoken

Discuss with a11y team: 

How should transcript navigation work on Safari with VoiceOver/ Android Chrome with Talkback? Having trouble emulating behavior of arrow key navigation.

https://corinagum.github.io/WebChat-release-testing/01.create-react-app/index.html

On browser: 
1. Type any activity in the sendbox
2. Press tab 2 times - should say 'end of transcript'
3. Can use arrow keys to navigate transcript

On mobile:
1. Type any activity in the sendbox
2. Turn on VoiceOver/Talkback
3. Single swipe left two times - notice that focus goes to empty "Connectivity status" container - it does not emulate tab behavior
4. Should we be changing rotor settings to emulate tabbing? If so, could you explain what settings to change to?

@Amit8527510735 @v-rampul 

Could you take a look at this behavior for us?

**Comment by corinagum** [comment]
@compulim scheduled time with a11y team for 3/11/2021

**Comment by compulim** [comment]
Discussed with accessibility team. The UX is great and should be certified as conformance after a change:

- On mobile device, we narrate "press arrow keys to navigate", we should remove this when on a mobile device because it is unlikely a mobile device has a keyboard with arrow keys

The condition to get this UX certified is that, "if the user can flick left/right to reach actionable buttons, even though the (activity) row is not selected, it is fine."

Additionally, for selecting the activity row, the user can use flick left/right to reach the row, and double tap to select the row.

Our work item is to remove the additional text "press arrow keys to navigate" when we are on a mobile device.

**Comment by corinagum** [comment]
Closing as dupe of #3758 :) (will add extra info there)
